cursorless-talon/src/modifiers/delimiter_range.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+from talon import Context, Module
+
+mod = Module()
+ctx = Context()
+
+
+delimiter_ranges = {
```

Comment:
These should be overridable by csv, no?

Reply:
Since the positions are not in csv yet either I think we should do both or neither.

Reply:
I'm a lot more confident about these. What do you expect to change here? These are fairly straightforward modifiers

Reply:
Nothing really but I think all of them should go in a position.csv file